src/software/SfM/main_openMVG2FBX.cpp

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2015 Stian Z. Vrba.
+//
+// // This Source Code Form is subject to the terms of the Mozilla Public
+// // License, v. 2.0. If a copy of the MPL was not distributed with this
+// // file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+#include <cstdlib>
+
+// Command line
+#include "third_party/cmdLine/cmdLine.h"
+
+// OpenMVG
+#include "openMVG/sfm/sfm.hpp"
+#include "openMVG/numeric/numeric.h"
+#include "openMVG/geometry/pose3.hpp"
+using namespace openMVG::sfm;
+
+// FBX
+#include <fbxsdk.h>
+
+int main(int argc, char **argv) {
+   
+    // Get arguments
+    CmdLine cmdLine;
+    std::string sfmDataFilename;
+    std::string sOutFBX = "";
+
+    cmdLine.add(make_option('i', sfmDataFilename, "sfmdata"));
+    cmdLine.add(make_option('o', sOutFBX, "outfile"));
+
+    try {
+        if (argc < 4) throw std::string("Invalid number of parameters in the command line.");
+        cmdLine.process(argc, argv);
+    } catch(const std::string &s) {
+        std::cout << "openMVG to FBX\n";
+        std::cout << "Usage: " << argv[0] << '\n'
+        << "[-i|--sfmdata filename, the SfM_Data file to convert]\n"
+        << "[-o|--outfile path]\n"
+        << std::endl;
+        std::cerr << s << std::endl;
+        return EXIT_FAILURE;
+    }
+
+    //
+    SfM_Data sfm_data;
+    if (!Load(sfm_data, sfmDataFilename, ESfM_Data(ALL))) {
+        std::cout << "Error: The input project file \""
+                  << sfmDataFilename << "\" cannot be read." << std::endl;
+        return EXIT_FAILURE;
+    }
+
+    //=================================================================================
+    // Create scene and mesh
+    //=================================================================================
+    
+    FbxAutoDestroyPtr<FbxManager> fbxmanager(FbxManager::Create());
+    FbxScene* fbxscene = FbxScene::Create(fbxmanager, "openMVG");
+    
+    FbxNode* fbxnode = FbxNode::Create(fbxscene, "sfm_node");
+    fbxscene->GetRootNode()->AddChild(fbxnode);
+    
+    FbxMesh* fbxmesh = FbxMesh::Create(fbxscene, "sfm_mesh");
+    fbxnode->SetNodeAttribute(fbxmesh);
+    
+    //=================================================================================
+    // Add points.  FBX has no native support for point clouds, so we create a mesh
+    // containing only vertices, w/o any other geometry.
+    //=================================================================================
+
+    {
+      const int point_count = sfm_data.GetLandmarks().size();
+      fbxmesh->InitControlPoints(point_count);
+      FbxVector4* out = fbxmesh->GetControlPoints();
+      for (const auto& lm: sfm_data.GetLandmarks()) {
+        const openMVG::Vec3 &pt = lm.second.X;
+        *out++ = FbxVector4(pt[0], pt[1], pt[2]);
```

Comment:
It's easy, so I'll do it :) A question though: are rgb components floats normalized to [0,1]?